plugins/mcreator-core/blockly/js/mcreator_extensions.js

```diff
@@ -282,6 +282,38 @@ function checkIfAfter(block, predicate) {
     return null;
 }
 
+Blockly.Extensions.registerMixin('check_duplicate_input_blocks',

```

Comment:
Validator/extension name should now be something more specific, mentioning type

Reply:
I renamed it to `disable_duplicate_input_type`. Is it better?